src/commands/wallet/command.ts

```diff
@@ -0,0 +1,3 @@
+import Program from "../../program.js";
+
+export default Program.command("wallet").description("Manage wallet related features for L2 and L1");
```

Comment:
For now there is no L1 features.
So we should either not mention L1 and just say `Manage zkSync wallet related features`
or
Add possibility to choose L1 network in the list. For example:
```
❯ zkSync Era Testnet 
  zkSync Era Mainnet 
  Local Dockerized node
  ---
  Ethereum Mainnet
  Ethereum Goerli Testnet
```